Textile/Classes/DataCallback.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+#import <Mobile/Mobile.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface DataCallback : NSObject<MobileDataCallback>
+
+- (instancetype)initWithCompletion:(void (^)(NSData*, NSString*, NSError*))completion;
```

Comment:
@sanderpick I thought we had discussed a data callback passing in the base64 string that includes the prefix? So what is this `NSData`?

Reply:
Oh no, the idea was to not send base64 encoded data right? Some consumers might want the raw and its way enough for them to encode it. For rendering the data as a data URL, I’m passing the media type so that the consumer just has to do the prefix concatenation after base64 encoding the data.

Reply:
Ah ok, that's great. That is great, just needed to verify that was intended.